dss_datamover/utils/config.py

```diff
@@ -57,7 +57,8 @@ def process_config(self):
             config = json.loads(cfg.read().decode('UTF-8', "ignore"))
         if self.params:
```

Comment:
You can check if **compaction** is present in config. If not, set it to True/yes. Rest will be to override the option if the user has explicitly given